apps/asap-server/src/entities/group.ts

```diff
@@ -28,14 +30,22 @@ export const parseGraphQLGroup = (item: GraphqlGroup): GroupResponse => {
     }),
   );
 
+  const tools: GroupTools = item.flatData?.tools ? item.flatData?.tools : {};
+
+  if (item.flatData?.calendars?.length) {
+    const url = new URL('https://calendar.google.com/calendar/r');
```

Comment:
Bit awkward that we do this in the backend here, but in the frontend for the calendar section

Reply:
Agree that it may be awkward however we want to have a clean interface on tools. I don't want to revisit this component if we decide that we the google calendar isn't based on an id :|

Reply:
Happy to change @fampinheiro ?

Reply:
The comment was more in the "keep it as it is" direction.